media/js/src/CollectionTab.jsx

```diff
@@ -135,8 +135,16 @@ export default class CollectionTab extends React.Component {
 
         return (
             <div role="tabpanel">
-                <h1 className="page-title">Collection</h1>
 
+                <div className="d-flex justify-content-between align-items-center flex-wrap">
+
+                    <h1 className="page-title">Collection</h1>
+
+                    <div className="btn btn-outline-secondary" aria-label="Add to Collection">
```

Comment:
The `className="btn btn-outline-secondary" aria-label="Add to Collection"` should go on the anchor tag below.

Reply:
Ah yes!